src/views/Login.vue

```diff
@@ -302,8 +303,12 @@ export default defineComponent({
         }
       }
 
+      if(isMaargLoginRequired(this.authStore.getRedirectUrl) && this.authStore.getMaargOms) {
+        omsRedirectionUrl = this.authStore.getMaargOms
+      }
+
       omsUrl = omsUrl ? omsUrl : this.authStore.oms.startsWith('http') ? this.authStore.oms.includes('/api') ? this.authStore.oms : `${this.authStore.oms}/api/` : this.authStore.oms
-      window.location.href = `${this.authStore.getRedirectUrl}?oms=${omsUrl}&token=${this.authStore.token.value}&expirationTime=${this.authStore.token.expiration}${isMaargLogin(this.authStore.getRedirectUrl) ? '&omsRedirectionUrl=' + this.authStore.oms : ''}`
+      window.location.href = `${this.authStore.getRedirectUrl}?oms=${omsUrl}&token=${this.authStore.token.value}&expirationTime=${this.authStore.token.expiration}${omsRedirectionUrl ? '&omsRedirectionUrl=' + omsRedirectionUrl : ''}`
```

Comment:
Check on the case when we are login in the marrg app, then how the omsRedirectionUrl parameter is passed, I think in that case no url will be passed resulting in login fail in the app.